CustomItems/Items/Scp5355.cs

```diff
@@ -76,12 +83,19 @@ protected override void UnsubscribeEvents()
             base.UnsubscribeEvents();
         }
 
+        /// <inheritdoc/>
         protected override void OnChanging(ChangingItemEventArgs ev)
         {
             if (ev.Player.Ammo[(int)AmmoType.Nato9] == 0)
             ev.Player.Ammo[(int)AmmoType.Nato9] = 1;
         }
 
+        /// <inheritdoc/>
+        protected override void OnPickingUp(PickingUpItemEventArgs ev)
+        {
+            playerMode.Add(ev.Player, 1);
```

Comment:
if (!playerMode.ContainsKey(ev.Player)
Otherwise you will cause an exception of a player picks up the item multiple times in a single round.